src/main/java/com/capitalone/dashboard/collector/DefaultArtifactoryClient.java

```diff
@@ -197,7 +197,9 @@ public DefaultArtifactoryClient(ArtifactorySettings artifactorySettings, RestCli
 							}
 
 							// find existing base artifact matching artifact item unique options
-							BaseArtifact suspect = baseArtifacts.stream().filter(b -> b.getArtifactItem().getArtifactName().equalsIgnoreCase(artifactItem.getArtifactName()) && b.getArtifactItem().getRepoName().equalsIgnoreCase(artifactItem.getRepoName())
+							BaseArtifact suspect = baseArtifacts.stream().filter(b ->
+									((String)b.getArtifactItem().getArtifactName()).equalsIgnoreCase((String)artifactItem.getArtifactName())

```

Comment:
can you use StringUtils here, its more cleaner that way. Additionally can you upgrade the version of core so that latest version of restclient is included as well as db timeout changes

Reply:
@rvema updated as suggested